pkg/gce-pd-csi-driver/node.go

```diff
@@ -320,25 +317,25 @@ func (ns *GCENodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpa
 }
 
 func (ns *GCENodeServer) GetVolumeLimits() (int64, error) {
```

Comment:
This may be better as a new file, maybe "gce-util.go" under https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/tree/master/pkg/gce-cloud-provider/compute.

Reply:
Move the function `GetVolumeLimits()` to a new file under the package of gcecloudprovider? Then do we need to pass the `MetadataService` as a parameter?

Reply:
Oh I see.  Ok it is fine where it is